Source/WebCore/DerivedSources.make

```diff
@@ -1755,6 +1755,7 @@ POSSIBLE_ADDITIONAL_MODERN_MEDIA_CONTROLS_STYLE_SHEETS = \
             $(ADDITIONS_PATH)/$(STYLE_SHEET)))))
 
 MODERN_MEDIA_CONTROLS_STYLE_SHEETS = \
+    $(WebCore)/Modules/modern-media-controls/controls/abstract-slider.css \
```

Comment:
I think you need to also add this to `Source/WebCore/CMakeLists.txt` and `Source/WebCore/WebCore.xcodeproj/project.pbxproj`.